src/runners/helpers/auth.py

```diff
@@ -29,13 +30,29 @@ def load_pkb(p8_private_key: bytes, passphrase: Optional[bytes]) -> bytes:
         password=ptpass.encode() if ptpass else None,
         backend=default_backend()
     )
+
     return private_key.private_bytes(
         encoding=serialization.Encoding.DER,
         format=serialization.PrivateFormat.TraditionalOpenSSL,
         encryption_algorithm=serialization.NoEncryption()
     )
 
 
+def load_pkb_rsa(p8_private_key: bytes, passphrase: Optional[bytes]) -> _RSAPrivateKey:
```

Comment:
use this in `load_pkb` instead of repeating the code

Reply:
done!